### DISCUSSION TITLE
JSX element type 'Card' does not have any construct or call signatures

### DISCUSSION THREAD

**Discussion by GautierBlandin** [ant-design/ant-design#31146] [Q&A] [answered]
Hey guys,

I'm starting to play around with ant-design, and so far it works great. However, I can't seem to use the Card element from the antd library. When I try to import it and use it in my code, I code this error : "JSX element type 'Card' does not have any construct or call signatures".

Anyone knows how to work around that ?

Thanks in advance :)

**Comment by GautierBlandin** [comment, marked as answer]
Answer for anyone who seems to encounter the same bug and can't understand what is happening : 
In my case, I simply didn't properly add the antd package (did yarn add in the wrong directory). So try installing antd design from the root directory of your project, and the bug should be resolved.

**Reply by AramZakaryan** [reply]
The same thing happened, could not understand the reason. thanks !
